pipelines/smartseq2_multisample/Loom_schema.md

```diff
@@ -0,0 +1,116 @@
+# What's in the Smart-seq2 Multi Sample Pipeline Loom File?
+
+The Loom file is an HDF5 file generated using [Loompy v.3.0.6](http://loompy.org/). The Multi Sample Loom is an aggregate of the output Loom files from the [Smart-seq2 Single Sample pipeline](../smartseq2_single_sample/). It contains global attributes, including the information about the batch (`plateid`; [Table 1](#table-1-global-attributes)). The main matrix contains the calculated TPMs from the [RSEM software](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-323) and there is an additional layer containing RSEM expected_counts (named "estimated_counts" in the Loom). The Loom additionally contains multiple metrics for both individual cells (the columns of the matrix; [Table 2](#table-2-column-attributes-cell-metrics)) and individual genes (the rows of the matrix; [Table 3](#table-3-row-attributes-gene-metrics)). The tables below document these metrics, list which tools generate them, and define them. This Loom file is the default matrix output of the Smart-seq2 Multi Sample pipeline.  
+
+**Note**: Loom files generated by the Smart-seq2 Multi Sample are different from the final Loom file distributed on the [Human Cell Atlas Data Portal](https://data.humancellatlas.org/explore/projects), which removes some of the metadata detailed in this document and contains additional metadata relating to each individual project. 
+
+## Table 1. Global Attributes
+The global attributes in the Loom apply to the whole file, not any specific part. There are three global attributes for the Multi Sample Smart-seq2 Loom. 
+
+| Attribute | Details |
+| :-- | :-- |
+| LOOM_SPEC_VERSION | String with the loom file spec version |
+| CreationDate | Date Loom file was generated |
+| sample_id | The plateid listed in the pipeline configuration file; by default it is set to the batch_id. |
+ 
+
+## Table 2. Column Attributes (Cell Metrics) 
+
+| Cell Metrics | Program |Details |
+|:---|:---:|:---| 
+| `ACCUMULATION_LEVEL` | [Picard](https://broadinstitute.github.io/picard/picard-metric-definitions.html) | See [Picard](https://broadinstitute.github.io/picard/picard-metric-definitions.html) documentation |
```

Comment:
Picard metrics documentation is approx. 3 years out of date. We don't have anything more up-to-date to link to though, so this is probably fine for now.